microhapulator/cli/pipe.py

```diff
@@ -187,6 +187,12 @@ def subparser(subparsers):
         help=SUPPRESS,
         # Hidden option for testing purposes
     )
+    cli.add_argument(
+        "--hspace",
+        metavar="HS",
+        default=-0.7,
+        help="Hhorizontal spacing between samples in the read distribution length ridge plots; negative value for this parameter enables overlapping plots; HS=-0.7 by default",
+    )
```

Comment:
New command line parameter to control the spacing between each sample in the ridge plot.

Reply:
Typo: Hhorizontal --> horizontal

Reply:
Whoops. This was me knowing that I should start with a lowercase letter but not being able to resist the urge to start with an uppercase letter 😆